plugins/importexport/doaj/DOAJExportDom.inc.php

```diff
@@ -1,435 +0,0 @@
-			case "lu": return "lub"; break;
-			case "lg": return "lug"; break;
-			case "mk": return "mac"; break;
-			case "mkd": return "mac"; break;
-			case "mh": return "mah"; break;
-			case "ml": return "mal"; break;
-			case "mi": return "mao"; break;
-			case "mri": return "mao"; break;
-			case "mr": return "mar"; break;
-			case "ms": return "may"; break;
-			case "msa": return "may"; break;
-			case "mg": return "mlg"; break;
-			case "mt": return "mlt"; break;
-			case "mo": return "mol"; break;
-			case "mn": return "mon"; break;
-			case "na": return "nau"; break;
-			case "nv": return "nav"; break;
-			case "nr": return "nbl"; break;
-			case "nd": return "nde"; break;
-			case "ng": return "ndo"; break;
-			case "ne": return "nep"; break;
-			case "nn": return "nno"; break;
-			case "nb": return "nob"; break;
-			case "no": return "nor"; break;
-			case "ny": return "nya"; break;
-			case "oc": return "oci"; break;
-			case "oj": return "oji"; break;
-			case "or": return "ori"; break;
-			case "om": return "orm"; break;
-			case "os": return "oss"; break;
-			case "pa": return "pan"; break;
-			case "pi": return "pli"; break;
-			case "pl": return "pol"; break;
-			case "pt": return "por"; break;
-			case "ps": return "pus"; break;
-			case "qu": return "que"; break;
-			case "rm": return "roh"; break;
-			case "ro": return "rum"; break;
-			case "ron": return "rum"; break;
-			case "rn": return "run"; break;
-			case "ru": return "rus"; break;
-			case "sg": return "sag"; break;
-			case "sa": return "san"; break;
-			case "sr": return "scc"; break;
-			case "srp": return "scc"; break;
-			case "si": return "sin"; break;
-			case "sk": return "slo"; break;
-			case "slk": return "slo"; break;
-			case "sl": return "slv"; break;
-			case "se": return "sme"; break;
-			case "sm": return "smo"; break;
-			case "sn": return "sna"; break;
-			case "sd": return "snd"; break;
-			case "so": return "som"; break;
-			case "st": return "sot"; break;
-			case "es": return "spa"; break;
-			case "sc": return "srd"; break;
-			case "ss": return "ssw"; break;
-			case "su": return "sun"; break;
-			case "sw": return "swa"; break;
-			case "sv": return "swe"; break;
-			case "ty": return "tah"; break;
-			case "ta": return "tam"; break;
-			case "tt": return "tat"; break;
-			case "te": return "tel"; break;
-			case "tg": return "tgk"; break;
-			case "tl": return "tgl"; break;
-			case "th": return "tha"; break;
-			case "ti": return "tir"; break;
-			case "to": return "ton"; break;
-			case "tn": return "tsn"; break;
-			case "ts": return "tso"; break;
-			case "tk": return "tuk"; break;
-			case "tr": return "tur"; break;
-			case "tw": return "twi"; break;
-			case "ug": return "uig"; break;
-			case "uk": return "ukr"; break;
-			case "ur": return "urd"; break;
-			case "uz": return "uzb"; break;
-			case "ve": return "ven"; break;
-			case "vi": return "vie"; break;
-			case "vo": return "vol"; break;
-			case "wa": return "wln"; break;
```

Comment:
This is ISO639-2b, right? Because we do not use language metadata filed any more i.e. here not every language is possible any more, but just article locale i.e. a locale that we have listed in `registry/locales.xml`, I could use the function `PKPLocale::get3LetterIsoFromLocale` i.e. the code from `registry/locales.xml`. Is this correct?